Tests/Fixtures/Configs/config-with-env-placeholder.yml

```diff
@@ -0,0 +1,18 @@
+output_dir: ${SWIFTGEN_OUTPUT_DIR}
+strings:
+  inputs: Sources1/Folder
```

Comment:
I'd add another env var here (typically `${PROJECT_DIR}`) or somewhere else in the YAML, so that the test will also ensure that all variables would be expanded, not just the first one